### ISSUE TITLE
Quick Search on Everywhere

### ISSUE THREAD

**Issue by kemal-atila** [Gisto/Gisto#190]
Hi,

Is there any global quick search shortcut button combination for easily copy and paste from all programs.

**Comment by sanusart** [comment]
@kemal-atila Do you mean something like OS X spotlight? Or internal in-program search?

**Comment by kemal-atila** [comment]
yes, something like spotlight or this one https://www.launchy.net/.

**Comment by sanusart** [comment]
We don't currently have something like that. Not sure how launchy works, but spotlight for instance, requires physical files on the filesystem in order to index/re-index.

**Comment by kemal-atila** [comment]
I mean opens a frame which contains auto completed textbox (only for gists ). Probably doesn't matter where do you store gists on filesystem or database,

**Comment by sanusart** [comment]
Inside the Gisto app?

**Comment by kemal-atila** [comment]
For example, pressed "Left Alt" and "Space" together when I was on desktop and than opened a new frame.

Its just a new feature suggestion :)
